### ISSUE TITLE
Bug: "ng-recaptcha3" has an incompatible peer dependency

### ISSUE THREAD

**Issue by Sampath-Lokuge** [armenstepanyan/ng-recaptcha3#43]
Hi,

Is this working with the latest Angular versions? It gives this error for me:

```
 Package "ng-recaptcha3" has an incompatible peer dependency to "@angular/common" (requires "^8.2.14" (extended), 
would install "14.2.12").
                  Package "ng-recaptcha3" has an incompatible peer dependency to "@angular/core" (requires "^8.2.14" (extended), would install "14.2.12").
× Migration failed: Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
  See "C:\Users\Sampath\AppData\Local\Temp\ng-cfXVBa\angular-errors.log" for further details.
```

**Comment by armenstepanyan** [comment]
Fixed in version ng-recaptcha3@1.4.0

**Comment by Sampath-Lokuge** [comment]
Thanks!
